hoot-js/src/main/cpp/hoot/js/schema/OsmSchemaJs.cpp

```diff
@@ -76,6 +81,10 @@ void OsmSchemaJs::Init(Handle<Object> exports)
               FunctionTemplate::New(current, isAncestor)->GetFunction());
   schema->Set(String::NewFromUtf8(current, "isArea"),
```

Comment:
Added all the crit methods needed here to support the new generic geometry conflate scripts. Also, the `isSpecificallyConflatable` method lets you know if any non-generic geometry (custom) conflation routine is capable of conflating a feature. If so, we know to skip such features in the generic geometry scripts so we don't end up with overlapping matches.